RadixWallet/Features/AppFeature/Overlay/FullScreenOverlayCoordinator+View.swift

```diff
@@ -13,9 +13,7 @@ extension FullScreenOverlayCoordinator {
 
 		public var body: some SwiftUI.View {
 			NavigationStack {
-				ZStack {
-					root(for: store.scope(state: \.root, action: { .child(.root($0)) }))
-				}
+				root(for: store.scope(state: \.root, action: { .child(.root($0)) }))
```

Comment:
ah, you can probably use `\.child.root` here

Reply:
right